generate/template/astronomy.py

```diff
@@ -2350,7 +2350,7 @@ def _QuadInterp(tm, dt, fa, fm, fb):
     df_dt = (2*Q*x + R) / dt
     return (t, df_dt)
 
-def Search(func, context, t1, t2, dt_tolerance_seconds):
+def Search(func: Callable[[object, Time], float], context: object, t1: Time, t2: Time, dt_tolerance_seconds: float) -> Optional[Time]:
```

Comment:
Hey Don, wanted to confirm if this is fine. The documentation here states that T1 and T2 are floats but the tests show that they're both `Time`.

Reply:
Yes, you are right. Another good catch! The documentation is wrong. It should say t1 and t2 are Time objects. Feel free to correct the documentation as part of your PR.